### PR TITLE
Maneesha/pkx 605 lilac upgrade

### PR DESCRIPTION
**Pull request by maneeshatejani1**
platform related changes for lilac upgrade

### CODE REVIEW

File: lms/envs/devstack.py [hunk 1]
```diff
@@ -82,6 +82,11 @@
     'debug_toolbar.middleware.DebugToolbarMiddleware',
 ]
 
+################################ DJANGO SILK ################################
+# if DEBUG:
```
Comment (emzubair) [inline lms/envs/devstack.py:86]: un-comment these lines

File: requirements/edx/coverage.in [hunk 2]
```diff
@@ -13,4 +13,4 @@
 -c ../constraints.txt
 
 coverage                            # Code coverage testing for Python
-diff-cover                          # Automatically find diff lines that need test coverage
+diff-cover==4.2.3                      # Automatically find diff lines that need test coverage
```
Comment (emzubair) [inline requirements/edx/coverage.in:16]: we don't need to pin it, go with default lilac changes.

File: requirements/edx/development.txt [hunk 3]
```diff
@@ -69,6 +73,7 @@ django-cors-headers==2.5.3  # via -c requirements/edx/../constraints.txt, -r req
 django-countries==5.5     # via -c requirements/edx/../constraints.txt, -r requirements/edx/testing.txt, edx-enterprise
 django-crum==0.7.9        # via -r requirements/edx/testing.txt, edx-django-utils, edx-enterprise, edx-proctoring, edx-rbac, edx-toggles, super-csv
 django-debug-toolbar==3.2.1  # via -r requirements/edx/development.in
+django-silk==4.1.0  # via -r requirements/edx/development.in
```
Comment (emzubair) [inline requirements/edx/development.txt:76]: do check django-silk version for python 3.8, and update accordingly

**Review by emzubair — CHANGES_REQUESTED** [review]

File: lms/envs/devstack.py (hunk 1 above)
Comment (maneeshatejani1) [reply lms/envs/devstack.py]: DONE

File: requirements/edx/coverage.in (hunk 2 above)
Comment (maneeshatejani1) [reply requirements/edx/coverage.in]: done

File: requirements/edx/development.txt (hunk 3 above)
Comment (maneeshatejani1) [reply requirements/edx/development.txt:76]: the current one already is as per python 3.8

**Review by emzubair — APPROVED** [review]
👍🏽

**Pull request merged**